packages/peregrine/lib/talons/Cms/useCmsPage.js

```diff
@@ -52,22 +52,9 @@ export const useCmsPage = props => {
     const cmsPage = data ? data.cmsPage : null;
     const rootCategoryId = data ? data.storeConfig.root_category_id : null;
 
-    // Only render <RichContent /> if the page isn't empty and doesn't contain
-    // the default CMS Page text. We do this so there is at least a useable home
-    // page by default, the category list component.
-    const hasContent = useMemo(() => {
-        return (
-            cmsPage &&
-            cmsPage.content &&
-            cmsPage.content.length > 0 &&
-            !cmsPage.content.includes('CMS homepage content goes here.')
-        );
-    }, [cmsPage]);
-
     return {
         cmsPage,
         error,
-        hasContent,
         rootCategoryId,
```

Comment:
`error` and `rootCategoryId` does not seem like they are used anymore.

Reply:
removed them.